src/bin/pg_autoctl/pgctl.c

```diff
@@ -271,14 +267,15 @@ pg_include_config(const char *configFilePath,
 	}
 
 	/* find the include 'postgresql-auto-failover.conf' line */
-	includeLine = regexp_first_match(currentConfContents, configIncludeRegex);
-	if (includeLine != NULL)
+	includeLine = strstr(currentConfContents, configIncludeLine);
```

Comment:
this does not account for extra space after `include` would it be acceptable ?

Reply:
I think yes, because we want to find exactly the line we have added to the configuration file from pg_autoctl, which alway has the same amount of space in there.